player/low-latency-streaming/index.html

```diff
@@ -1,7 +1,7 @@
 <script type="text/javascript"
-        src="//cdn.bitmovin.com/player/web/8.4.0/bitmovinplayer.js"></script>
+        src="https://cdn.jsdelivr.net/npm/bitmovin-player@latest/bitmovinplayer.js"></script>
 <script type="text/javascript"
-        src="https://cdn.bitmovin.com/analytics/web/beta/2/bitmovinanalytics.min.js"></script>
+        src="https://cdn.jsdelivr.net/npm/bitmovin-analytics@latest/bitmovinanalytics.min.js"></script>
```

Comment:
I think we don't need to include analytics, as it comes with the player...

Reply:
removed